### ISSUE TITLE
Consider normalizing images using standard ImageNet Params

### ISSUE THREAD

**Issue by manuelli** [RobotLocomotion/pytorch-dense-correspondence#80]
The pytorch [docs](http://pytorch.org/docs/master/torchvision/models.html) recommend normalizing the images with

```
normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],
                                 std=[0.229, 0.224, 0.225])
```

when using models pre-trained on ImageNet. Currently we are computing dataset specific mean/var
